python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -1152,23 +1149,23 @@ def measure(self, measRecord, exposure, posExp=None, negExp=None):
 
         # add chi2, coord/flux uncertainties (TBD), dipole classification
         # Add the relevant values to the measRecord
-        measRecord[self.posFluxKey] = result.posFlux
-        measRecord[self.posFluxErrKey] = result.signalToNoise   # to be changed to actual sigma!
-        measRecord[self.posCentroidKeyX] = result.posCentroidX
-        measRecord[self.posCentroidKeyY] = result.posCentroidY
+        measRecord[self.posFluxKey.getInstFlux()] = result.posFlux
```

Comment:
I didn't realize this was how we needed to set by key!

Reply:
This is one way. I think when this was written, the string-based method (`measRecord["blah_instFlux"]`) was much slower, which has since been mostly fixed. But having the keys in the plugin makes it easier, since you can just get the field key directly.